community/kernel/pom.xml

```diff
@@ -143,6 +143,9 @@ the relevant Commercial Agreement.
             <include>org/neo4j/procedure/*</include>
             <include>org/neo4j/helpers/collection/*</include>
           </includes>
+          <excludes>
+            <exclude>org/neo4j/unsafe/batchinsert/DirectRecordAccess</exclude>
```

Comment:
We should probably exclude `org.neo4j.unsafe.batchinsert.DirectRecordAccessSet` too while we're at it. Or move them to the `internal` sub-package.